docs/contivctl.md

```diff
@@ -0,0 +1,226 @@
+## contivctl
+
+contivctl command line tool lets you add/modify/delete objects into contiv object model using REST api.
+
+Command can be found under $GOPATH/bin directory (/opt/gopath/bin in vagrant VMs) after a build. 
```

Comment:
suggest to add a **Note:** tag before this line so that it get's necessary attention of the reader.